.editorconfig

```diff
@@ -21,5 +25,5 @@ indent_size = 2
 [*.md]
 trim_trailing_whitespace = false
 
-[*.txt]
-end_of_line = crlf
+[LICENSE]
+indent_style = unset
```

Comment:
```suggestion
[*.txt]
end_of_line = crlf

[LICENSE]
indent_style = unset
```

Lets add the `[*.txt]` entry back in, so there's as few deviations from the WP editor config file as possible.

Reply:
That WordPress way resembles typewriters and line printers.